curvefs/src/tools/status/curvefs_copyset_status.cpp

```diff
@@ -41,46 +41,47 @@ void CopysetStatusTool::PrintHelp() {
 }
 
 int CopysetStatusTool::RunCommand() {
-    if (copyInfoListTool_->RunCommand() == 0) {
-        auto response = copyInfoListTool_->GetResponse();
-        std::map<
-            uint64_t,
-            std::vector<curvefs::metaserver::copyset::CopysetStatusResponse>>
-            key2Status;
-        copyset::CopysetInfo2CopysetStatus(*response.get(), &key2Status);
+    copyInfoListTool_->RunCommand();
+    auto response = copyInfoListTool_->GetResponse();
+    std::map<uint64_t,
+             std::vector<curvefs::metaserver::copyset::CopysetStatusResponse>>
+        key2Status;
+    copyset::CopysetInfo2CopysetStatus(*response.get(), &key2Status);
 
-        std::map<uint64_t, std::vector<curvefs::mds::topology::CopysetValue>>
-            key2Info;
+    std::map<uint64_t, std::vector<curvefs::mds::topology::CopysetValue>>
+        key2Info;
 
-        copyset::Response2CopysetInfo(*response.get(), &key2Info);
+    copyset::Response2CopysetInfo(*response.get(), &key2Info);
 
-        bool isHealth = true;
-        for (auto const& i : key2Info) {
-            if (copyset::checkCopysetHelthy(i.second, key2Status[i.first]) !=
-                copyset::CheckResult::kHealthy) {
-                isHealth = false;
-                break;
-            }
+    bool isHealth = true;
+    for (auto const& i : key2Info) {
+        if (copyset::checkCopysetHelthy(i.second, key2Status[i.first]) !=
+            copyset::CheckResult::kHealthy) {
+            isHealth = false;
```

Comment:
The unhealthy copysets can be recorded and show the detail if needed.

Reply:
> The unhealthy copysets can be recorded and show the detail if needed.

now is show all copyset, will add this feature.